packages/falter-common/files-common/sbin/uci2ffwizard

```diff
@@ -0,0 +1,145 @@
+#!/bin/sh
+
+#
+#   This script parses all important values into a json file. With this
+#   file one can restore the complete basic configuration of a freifunk
+#   berlin router.
+#
+#   It originates from Freifunk Berlin and is licensed under
+#   GNU General Public License v3.0 or later
+#   Copyright (C) 2023   Martin Hübner
+#
+
+# set -e
+# shellcheck shell=dash
+# shellcheck disable=SC2086
+# shellcheck disable=SC2154
+
+# shellcheck source=/dev/null
+. /lib/functions.sh
+# shellcheck source=/dev/null
+. /usr/share/libubox/jshn.sh
+
+# get contact and community data
+uci_load freifunk
+name="$(uci_get freifunk contact name)"
+nick="$(uci_get freifunk contact nickname)"
+mail="$(uci_get freifunk contact mail)"
+phone="$(uci_get freifunk contact phone)"
+homepage="$(uci_get freifunk contact homepage)" # whitespace-separated, with single quotes, if string contains whitspace
+loc_description="$(uci_get freifunk contact location)"
+com_name="$(uci_get freifunk community name)"
+
+# collect nodes location
+uci_load system
+longitude="$(uci_get system @system[-1] longitude)"
+latitude="$(uci_get system @system[-1] latitude)"
+
+# if contact field contains a config-wizard URL, write it into URL-field
+if case "$mail" in
+    https://config.berlin.freifunk.net/contact*) true ;;
+    *) false ;;
+esac then
+    url="$mail"
+    mail=""
+fi
+
+# load IP-Addresses
+uci_load ffwizard
+share="$(uci_get ffwizard settings sharenet)"
+download="$(( $(uci_get ffwizard settings usersBandwidthDown) * 1000))"
+upload="$(( $(uci_get ffwizard settings usersBandwidthUp) * 1000))"
```

Comment:
changed in new commit

Reply:
we need to make more changes here. If the bandwith is not set my solution will read it as 0, however it should be absent instead.

Reply:
We can't make the upload/download being absent, because it is required by the json-schema. If the internet-connection is defined, we need to whole connection being defined.